### PR TITLE
Updated README & Fix validIds.test.ts leading zeros test

### PR DESCRIPTION
**Pull request by Cactooz**
- Update README.md
  - Add how to create and add new songs as well as how to modify existing songs.
  - Move Project scripts bellow Song scripts.
  - Fix inconsistent `'` to `"`.
- Fix validIds.test.ts leading zeros test mapping to unused `files` variable.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,29 +150,7 @@ Make sure to make any relevant updates to the documentation.
...
-Runs tests to validate that the build is valid and up to date.
-
-#### `yarn lint`
-
-Checks that all files (code and songs) follows style rules.
-
-#### `yarn format`
-
-Formats all files (code and songs) to follow style rules.
+The scripts are divided into two sections: "project scripts" and "song scripts".
```
Comment (StelFoog) [inline README.md:153]:
```suggestion
The scripts are divided into two sections: "song scripts" and "project scripts".
```

**Review by StelFoog — CHANGES_REQUESTED** [review]
Mostly LGTM, only want a small change

File: README.md (hunk 1 above)
Comment (Cactooz) [reply README.md]: Resolved with [6246e3a](https://github.com/insektionen/songlist/pull/1/commits/6246e3a3b952f18b378d6db226545e227ec468ce).

File: README.md [hunk 2]
```diff
@@ -9,13 +9,15 @@ You can also find information on how to contribute to this repository, either by
 - [Contributing (general)](#contributing-general)
   - [Quick guide](#quick-guide)
 - [Contributing (songs)](#contributing-songs)
+  - [Format](#format)
+  - [Creating](#creating)
```
Comment (StelFoog) [inline README.md:13]: Should be `modifying` instead of `format` and should be in opposite order

**Review by StelFoog — CHANGES_REQUESTED** [review]
Small fix required

File: README.md (hunk 2 above)
Comment (Cactooz) [reply README.md]: Resolved with [f74c3a3](https://github.com/insektionen/songlist/pull/1/commits/f74c3a3ae334e4bc03f38374b34162988bce494f).

**Review by StelFoog — APPROVED** [review]
LGTM!

**Pull request merged**